client/state/reader/follows/reducer.js

```diff
@@ -179,15 +219,19 @@ export const items = createReducer(
 			};
 		},
 		[ READER_SUBSCRIBE_TO_NEW_POST_EMAIL ]: ( state, action ) =>
-			updatePostSubscription( state, action ),
+			updateEmailSubscription( state, action ),
```

Comment:
since all of these don't change the arguments, could these just be `[ READER_SUBSCRIBE_TO_NEW_POST_EMAIL ]: updateEmailSubscription`

Reply:
Much simpler 👍